compiler/AST/interfaces.cpp

```diff
@@ -71,6 +72,8 @@ DefExpr* InterfaceSymbol::buildDef(const char* name,
 
   if (gHashable == nullptr && name == astr("hashable")) {
     gHashable = isym;
+  } else if (gContextManager == nullptr && name == astr("contextManager")) {
+    gContextManager = isym;
   }
```

Comment:
Do the checks `gHashable == nullptr` and `gContextManager == nullptr` ensure that gHashable and gContextManager will refer to the predefined interfaces?

I think `!strcmp(name, "hashable")` is faster (and more direct) than `name == astr("hashable")`.

Reply:
> Do the checks gHashable == nullptr and gContextManager == nullptr ensure that gHashable and gContextManager will refer to the predefined interfaces?

Yup!